examples/gno.land/r/gnoland/hof/hof.gno

```diff
@@ -0,0 +1,174 @@
+// Package hof is the gno.land hall of fame realm.
+// It contains a permanent exhibition of items in the hall of fame,
+// a possibility to host temporary ones created by its admins.
+package hof
+
+import (
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/seqid"
+	"gno.land/p/demo/ufmt"
+)
```

Comment:
The admin should be able to pause the contract completely and also have the ability to delete some entries. Essentially, we allow it to run until it becomes spammed, at which point we clean and pause it. Later, we can revisit the hof contract to make it DAO-managed.

Reply:
good thing we have p/demo/pausable :)

Reply:
[a646518](https://github.com/gnolang/gno/pull/2842/commits/a64651897956d45980086aa2f9cd054f0a50d6ea)